SourceX/main.cpp

```diff
@@ -22,8 +22,20 @@ static std::string build_cmdline(int argc, char **argv)
 	return str;
 }
 
+void quit(){

```

Comment:
Please use the project code style https://github.com/diasurgical/devilution/wiki/Code-Style